.github/workflows/release.yml

```diff
@@ -116,6 +116,17 @@ jobs:
           cd scripts && ../ve1/bin/pip3 install -r requirements.txt && cd ..
           cd scripts && ../ve1/bin/python3 setup.py install && cd ..
 
+      - name: Run All Tests
```

Comment:
we don't want this here,  It needs to be in test.yml. Currently in test.yaml we decide not to run any tests for a release and then approve and merge the release PR. That needs to change so that it now does run tests for a release and does not merge and release if tests fail. The test then need to be the full bucket. This flow is about creating PR's to copy charts from the chart repo to dev repo and copy flows to charts repo.

Reply:
My bad, we do want it here but would good to have a few steps earlier to avoid the unnecessary checkout and setting up PR scripts again.

Reply:
Updated workflow to addressed this comment.